File: numberOfPeopleInBus.py
# file: 	numberOfPeopleInBus.cpp
# author:	Diogo Marques
# data:		13/04/2025
# description:
#	Number of People in Bus at codewars.com
def number(bus_stops):
    bus_count = 0
    for i in bus_stops:
        bus_count += i[0]
        bus_count += -i[1]
    return bus_count

File: numberOfPeopleInBus.cpp
// file: 	numberOfPeopleInBus.cpp
// author:	Diogo Marques
// data:	13/04/2025
// description:
//	Number of People in Bus at codewars.com
#include <utility>
#include <vector>

unsigned int number(const std::vector<std::pair<int, int>>& busStops){
  int bus_count = 0;
  for(int i = 0; i < busStops.size(); i++){
    bus_count += busStops[i].first;
    bus_count += - busStops[i].second;
  }
  return bus_count;
}